pkg/api/datasources.go

```diff
@@ -127,6 +128,9 @@ func (hs *HTTPServer) DeleteDataSourceById(c *models.ReqContext) response.Respon
 
 	err = hs.DataSourcesService.DeleteDataSource(c.Req.Context(), cmd)
 	if err != nil {
+		if errors.As(err, &secretsPluginError) {
```

Comment:
Hmm, using a pointer to pointer here and in other error checks doesn't look correct?

Reply:
Ah, sloppy by me. Fixed in [7a8b88c](https://github.com/grafana/grafana/pull/50542/commits/7a8b88cb5b8af0c9a07908c0aa7d99d68172e06b). Going to give this a final round of testing before I click merge. Thanks!